enterprise/security/src/main/java/org/neo4j/server/security/enterprise/configuration/SecuritySettings.java

```diff
@@ -129,7 +131,7 @@
             "The LDAP referral behavior when creating a connection. This is one of `follow`, `ignore` or `throw`.\n" +
```

Comment:
Would you advise to move it into a normal sentence without line breaks and asterisks then? It's easy to fix.